src/main/java/core/basesyntax/CircleSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class CircleSupplier {
```

Comment:
We don't need an exceptional circle supplier only for the default figure. Please call `Circle` constructor with default parameters where it's needed.